Description: add -std=gnu++03 and -fno-delete-null-pointer-checks to all build prefix files
 Firebird 3.0.3 adds these to several prefix files, but not all
 The Debian package stopped adding these flags "by hand", leading to several
 architectures failing to build Firebird 3.0.3:
  https://buildd.debian.org/status/fetch.php?pkg=firebird3.0&arch=arm64&ver=3.0.3.32900.ds4-1&stamp=1517766258&raw=0
  https://buildd.debian.org/status/fetch.php?pkg=firebird3.0&arch=armel&ver=3.0.3.32900.ds4-1&stamp=1517767263&raw=0
  https://buildd.debian.org/status/fetch.php?pkg=firebird3.0&arch=armhf&ver=3.0.3.32900.ds4-1&stamp=1517767152&raw=0
  https://buildd.debian.org/status/fetch.php?pkg=firebird3.0&arch=mips&ver=3.0.3.32900.ds4-1&stamp=1517767223&raw=0
  https://buildd.debian.org/status/fetch.php?pkg=firebird3.0&arch=mipsel&ver=3.0.3.32900.ds4-1&stamp=1517768560&raw=0
  https://buildd.debian.org/status/fetch.php?pkg=firebird3.0&arch=ppc64el&ver=3.0.3.32900.ds4-1&stamp=1517765884&raw=0
  https://buildd.debian.org/status/fetch.php?pkg=firebird3.0&arch=s390x&ver=3.0.3.32900.ds4-1&stamp=1517765834&raw=0
  https://buildd.debian.org/status/fetch.php?pkg=firebird3.0&arch=hppa&ver=3.0.3.32900.ds4-1&stamp=1517770256&raw=0
  https://buildd.debian.org/status/fetch.php?pkg=firebird3.0&arch=ia64&ver=3.0.3.32900.ds4-1&stamp=1517766796&raw=0
  https://buildd.debian.org/status/fetch.php?pkg=firebird3.0&arch=powerpc&ver=3.0.3.32900.ds4-1&stamp=1517765959&raw=0
  https://buildd.debian.org/status/fetch.php?pkg=firebird3.0&arch=ppc64&ver=3.0.3.32900.ds4-1&stamp=1517766499&raw=0
  https://buildd.debian.org/status/fetch.php?pkg=firebird3.0&arch=sparc64&ver=3.0.3.32900.ds4-1&stamp=1517766590&raw=0
Author: Damyan Ivanov <dmn@debian.org>

--- a/builds/posix/prefix.linux_ia64
+++ b/builds/posix/prefix.linux_ia64
@@ -18,7 +18,7 @@
 # 2 Oct 2002, Nickolay Samofatov - Major cleanup
 # Adriano dos Santos Fernandes - IA-64 port
 
-COMMON_FLAGS=-ggdb -DFB_SEND_FLAGS=MSG_NOSIGNAL -DLINUX -DIA64 -pipe -MMD -fPIC -fmessage-length=0
+COMMON_FLAGS=-ggdb -DFB_SEND_FLAGS=MSG_NOSIGNAL -DLINUX -DIA64 -pipe -MMD -fPIC -fmessage-length=0 -std=gnu++03 -fno-delete-null-pointer-checks
 OPTIMIZE_FLAGS=-O3 -fno-omit-frame-pointer 
 WARN_FLAGS=-Wall -Wno-switch -Wno-parentheses -Wno-unknown-pragmas -Wno-unused-variable -Wno-non-virtual-dtor
 
--- a/builds/posix/prefix.linux_mips
+++ b/builds/posix/prefix.linux_mips
@@ -1,4 +1,4 @@
-COMMON_FLAGS=-ggdb -DFB_SEND_FLAGS=MSG_NOSIGNAL -DLINUX -pipe -MMD -fPIC -fmessage-length=0
+COMMON_FLAGS=-ggdb -DFB_SEND_FLAGS=MSG_NOSIGNAL -DLINUX -pipe -MMD -fPIC -fmessage-length=0 -std=gnu++03 -fno-delete-null-pointer-checks
 OPTIMIZE_FLAGS=-O3 -fno-omit-frame-pointer -fno-builtin
 WARN_FLAGS=-Wall -Wno-switch -Wno-parentheses -Wno-unknown-pragmas -Wno-unused-variable -Wno-non-virtual-dtor
 
--- a/builds/posix/prefix.linux_mipsel
+++ b/builds/posix/prefix.linux_mipsel
@@ -21,5 +21,5 @@ LD=@CXX@
 
 # -fno-builtin is used because GCC 3.0-3.2.2 had bug with builtins expansion
 # you may remove it if engine is getting compiled with any other GCC version
-PROD_FLAGS=-ggdb -O3 -fno-omit-frame-pointer -DLINUX -pipe -MMD -fPIC
+PROD_FLAGS=-ggdb -O3 -fno-omit-frame-pointer -DLINUX -pipe -MMD -fPIC -std=gnu++03 -fno-delete-null-pointer-checks
 DEV_FLAGS=-ggdb -DLINUX -DDEBUG_GDS_ALLOC -pipe -MMD -p -fPIC -Wall -Wno-switch -Wno-non-virtual-dtor
--- a/builds/posix/prefix.linux_powerpc
+++ b/builds/posix/prefix.linux_powerpc
@@ -18,6 +18,6 @@
 #
 # 2 Oct 2002, Nickolay Samofatov - Major cleanup
 
-PROD_FLAGS=-ggdb -fno-omit-frame-pointer -fsigned-char -DLINUX -pipe -MMD -fPIC -fmessage-length=0 -DLINUX -DPowerPC -DPPC -O3
+PROD_FLAGS=-ggdb -fno-omit-frame-pointer -fsigned-char -DLINUX -pipe -MMD -fPIC -fmessage-length=0 -DLINUX -DPowerPC -DPPC -O3 -std=gnu++03 -fno-delete-null-pointer-checks
 DEV_FLAGS=-ggdb -O0 -DLINUX -pipe -MMD -p -fPIC -Wall -Wno-switch -Wno-parentheses -Wno-unknown-pragmas -Wno-unused-variable -fmessage-length=0 -fsigned-char -DLINUX -DPowerPC -DPPC -Wno-non-virtual-dtor
 #DEV_FLAGS=-DUSE_VALGRIND -ggdb -O0 -DLINUX -pipe -MMD -p -fPIC -Wall -Wno-switch -Wno-parentheses -Wno-unknown-pragmas -Wno-unused-variable -fmessage-length=0 -DLINUX -DPowerPC -DPPC -Wno-non-virtual-dtor
--- a/builds/posix/prefix.linux_powerpc64
+++ b/builds/posix/prefix.linux_powerpc64
@@ -23,5 +23,5 @@
 #PROD_FLAGS=-ggdb -O3 -fno-omit-frame-pointer -DLINUX -pipe -MMD -fPIC
 #DEV_FLAGS=-ggdb -DLINUX -DDEBUG_GDS_ALLOC -pipe -MMD -p -fPIC -Wall -Wno-switch
 
-PROD_FLAGS=-O3 -DLINUX -DPPC64 -pipe -p -MMD -fPIC -fsigned-char -fmessage-length=0
+PROD_FLAGS=-O3 -DLINUX -DPPC64 -pipe -p -MMD -fPIC -fsigned-char -fmessage-length=0 -std=gnu++03 -fno-delete-null-pointer-checks
 DEV_FLAGS=-ggdb -DLINUX -DPPC64 -pipe -p -MMD -fPIC -Wall -fsigned-char -fmessage-length=0 -Wno-non-virtual-dtor
--- a/builds/posix/prefix.linux_powerpc64el
+++ b/builds/posix/prefix.linux_powerpc64el
@@ -23,5 +23,5 @@
 #PROD_FLAGS=-ggdb -O3 -fno-omit-frame-pointer -DLINUX -pipe -MMD -fPIC
 #DEV_FLAGS=-ggdb -DLINUX -DDEBUG_GDS_ALLOC -pipe -MMD -p -fPIC -Wall -Wno-switch
 
-PROD_FLAGS=-O3 -DLINUX -DPPC64EL -pipe -p -MMD -fPIC -fsigned-char -fmessage-length=0
+PROD_FLAGS=-O3 -DLINUX -DPPC64EL -pipe -p -MMD -fPIC -fsigned-char -fmessage-length=0 -std=gnu++03 -fno-delete-null-pointer-checks
 DEV_FLAGS=-ggdb -DLINUX -DPPC64EL -pipe -p -MMD -fPIC -Wall -fsigned-char -fmessage-length=0 -Wno-non-virtual-dtor
--- a/builds/posix/prefix.linux_s390x
+++ b/builds/posix/prefix.linux_s390x
@@ -18,7 +18,7 @@
 #
 # 2 Oct 2002, Nickolay Samofatov - Major cleanup
 
-COMMON_FLAGS=-ggdb -DFB_SEND_FLAGS=MSG_NOSIGNAL -DLINUX -pipe -MMD -fPIC -fmessage-length=0 -fsigned-char
+COMMON_FLAGS=-ggdb -DFB_SEND_FLAGS=MSG_NOSIGNAL -DLINUX -pipe -MMD -fPIC -fmessage-length=0 -fsigned-char -std=gnu++03 -fno-delete-null-pointer-checks
 OPTIMIZE_FLAGS=-O3 -fno-omit-frame-pointer -fno-builtin
 WARN_FLAGS=-Wall -Wno-switch -Wno-parentheses -Wno-unknown-pragmas -Wno-unused-variable -Wno-non-virtual-dtor
 
--- a/builds/posix/prefix.linux_sparc32
+++ b/builds/posix/prefix.linux_sparc32
@@ -18,6 +18,6 @@
 #
 # 2 Oct 2002, Nickolay Samofatov - Major cleanup
 
-COMMON_FLAGS=-DLINUX -pipe -MMD -fPIC -Dsparc -DFB_SEND_FLAGS=MSG_NOSIGNAL
+COMMON_FLAGS=-DLINUX -pipe -MMD -fPIC -Dsparc -DFB_SEND_FLAGS=MSG_NOSIGNAL -std=gnu++03 -fno-delete-null-pointer-checks
 PROD_FLAGS=-ggdb -mcpu=ultrasparc -mtune=ultrasparc -O3 $(COMMON_FLAGS)
 DEV_FLAGS=-ggdb -p -Wall -Wno-switch  $(COMMON_FLAGS) -Wno-non-virtual-dtor
--- a/builds/posix/prefix.freebsd
+++ b/builds/posix/prefix.freebsd
@@ -20,5 +20,5 @@
 
 EXE_LINK_OPTIONS+=-Wl,-rpath,../gen/firebird/lib
 
-PROD_FLAGS=-O -fno-builtin -DFREEBSD -pipe -MMD -fPIC
-DEV_FLAGS=-ggdb -DFREEBSD -pipe -MMD -p -fPIC -Wall -Wno-non-virtual-dtor
+PROD_FLAGS=-O -fno-builtin -DFREEBSD -pipe -MMD -fPIC -std=gnu++03
+DEV_FLAGS=-ggdb -DFREEBSD -pipe -MMD -p -fPIC -Wall -Wno-non-virtual-dtor -std=gnu++03
--- a/builds/posix/prefix.freebsd_amd64
+++ b/builds/posix/prefix.freebsd_amd64
@@ -20,5 +20,5 @@
 
 EXE_LINK_OPTIONS+=-Wl,-rpath,../gen/firebird/lib
 
-PROD_FLAGS=-O -fno-builtin -DFREEBSD -DAMD64 -pipe -MMD -fPIC
-DEV_FLAGS=-ggdb -DFREEBSD -DAMD64 -pipe -MMD -p -fPIC -Wall -Wno-non-virtual-dtor
+PROD_FLAGS=-O -fno-builtin -DFREEBSD -DAMD64 -pipe -MMD -fPIC -std=gnu++03
+DEV_FLAGS=-ggdb -DFREEBSD -DAMD64 -pipe -MMD -p -fPIC -Wall -Wno-non-virtual-dtor -std=gnu++03
